src/Handler/StreamHandler.php

```diff
@@ -181,13 +185,26 @@ private function checkDecode(array $options, array $headers, $stream)
      *
      * @param StreamInterface $source
      * @param StreamInterface $sink
+     * @param null|int        $contentLength Amount of data to read.
```

Comment:
`getHeaderLine` returns a string in any case. so null is not used currently and int is not really true.